### ISSUE TITLE
Obtain Appsflyer credentials

### ISSUE THREAD

**Issue by sherifnada** [airbytehq/airbyte#4930]
## Tell us about the problem you're trying to solve
A contributor contributed the Appsflyer connector back in the day (singer based) but we have no way of testing it in CI and therefore have not published it. We should get a sandbox account for Appsflyer to be able to run CI on the connector.

**Comment by cheyura** [comment]
@sherifnada Here is the quote from support(when asking about sandbox):
"I will be happy to assist you. The answer is no. We allow our advertisers to use the production environment for both development and store builds. The best way to separate tests is testing through specific media source for tests versions. You may also download the raw report and filter by media source / app version."

**Comment by sherifnada** [comment]
@cheyura is it possible to get a production account that we populate with fake data then? Or is this basically saying "unless you have a production account and are running real campaigns, you can't test" ?

**Comment by cheyura** [comment]
> @cheyura is it possible to get a production account that we populate with fake data then? Or is this basically saying "unless you have a production account and are running real campaigns, you can't test" ?

@sherifnada yes,but again without real app we can nothing to do with prod account.However, I wrote application for partnership

**Comment by cheyura** [comment]
@sherifnada blocked since no way to create a sandbox and for prod App is needed

**Comment by sherifnada** [comment]
@cheyura i labelled it as blocked and moved to icebox column
